Dockerfile

```diff
@@ -0,0 +1,26 @@
+
+## Dependency Stage ##
+FROM node:18-alpine AS dep
+WORKDIR /app
+COPY package.json ./package.json
+COPY yarn.lock ./yarn.lock
+# copy `dist` if it exists already
+COPY dis[t]/web ./dist/
+COPY node_module[s] ./node_modules
```

Comment:
> No worries and thanks for looking at my commit! Your edit looks good! I did not think about skipping install or build if node_modules and/or dist already exist as there might be conflict between different OS/kernel and architecture, but since Node is multi-arch, I do not think it matters at all?

Now that you have mentioned it, I actually think it makes more sense to skip copying the node_modules to the CI step for that reason. _(some node deps do install different things for different architectures)_
![image](https://github.com/MarmadileManteater/FreeTubeCordova/assets/106682128/ebdbfe1c-953e-49c8-9f09-1f6087137354)

```suggestion
```